vendor/engines/secure_rooms/app/models/secure_rooms/card_reader.rb

```diff
@@ -4,6 +4,9 @@ class CardReader < ActiveRecord::Base
 
     belongs_to :secure_room, foreign_key: :product_id
 
+    validates :card_reader_number, :control_device_number, presence: true
+    validates :card_reader_number, uniqueness: true
```

Comment:
I don't think so, I would assume this is like a serial number and thus unique to every reader, did we figure this out though at some point?